frontend/src/scenes/plugins/Plugins.tsx

```diff
@@ -40,7 +40,12 @@ export function Plugins({ user }: { user: UserType }): JSX.Element | null {
                         </sup>
                     </>
                 }
-                caption="Plugins enable you to extend PostHog's core data processing functionality."
+                caption={
+                    <>
+                        Plugins enable you to extend PostHog's core data processing functionality. You can also{' '}
```

Comment:
My suggestion for the blurb after the first sentence would be "Use verified plugins from the [official repository](), or [build your own]()."

Reply:
On cloud atleast, there's no option to "use verified plugins from the official repository" (nor to build for that matter lol). I think the goal behind this is to encourage building your own plugins?
Since everything in the official repository is already available as an option to install.

Reply:
@Twixes - are you ok with this?

Reply:
Well, on Cloud you can use repository plugins, just not freely because we control versions (and a few are not public _yet_). Right that building is definitely not possible on Cloud yet though, maybe this should have a ternary operator based on `preflight.cloud`?
For me it's just that "You can also" sounds a bit awkward without an alternative version.
But anyway this is not a blocker at all! Always easy to adjust copy in another PR.